identity-iota/src/tangle/message_index.rs

```diff
@@ -143,55 +142,55 @@ mod tests {
   }
 
   #[rustfmt::skip]
```

Comment:
these `#[rustfmt::skip]` attributes should have been removed a long time ago. probably best to do now